decidim-core/app/cells/decidim/content_blocks/sub_hero_cell.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Decidim
+  module ContentBlocks
+    class SubHeroCell < Decidim::ViewModel
+      include Decidim::IconHelper
+      include Decidim::SanitizeHelper
+
+      def show
+        return if translated_attribute(current_organization.description).blank?
+        render
+      end
+
+      delegate :current_organization, to: :controller
```

Comment:
Maybe we can add this to `Decidim::ViewModel`?

Reply:
I won't do it by now as it might affect other cells.